package.json

```diff
@@ -14,21 +14,22 @@
     "@types/node": "20.4.5",
     "@types/react": "18.2.17",
     "@types/react-dom": "18.2.7",
-    "autoprefixer": "^10.4.12",
```

Comment:
Why would we change the `autoprefixer` & `tailwindcss` packages to dev dependencies?

Reply:
Woops, I think i accidentally installed a new version of the tailwind css, when i was trying to figure out how to do styling for the dashboard! Sorry about that, i will update it back to how it was originally but still keeping the same styles!